cypress/integration/pages/homePage/tests.js

```diff
@@ -0,0 +1,8 @@
+export default ({ service, pageType }) => {
+  it.skip(`should render a simple ${service} ${pageType} page`, () => {
```

Comment:
This test is now failing since we merged #10706. We should decide what types of tests we need for the HomePage, now that it is rendering curations, similar to the Topic Page